opentracing-web-servlet-filter/src/test/java/io/opentracing/contrib/web/servlet/filter/AbstractJettyTest.java

```diff
@@ -182,4 +184,15 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         @Override
         public void destroy() {}
     }
+
+    public static class CurrentSpanServlet extends HttpServlet {
+
+        @Override
+        public void doGet(HttpServletRequest request, HttpServletResponse response)
+                throws ServletException, IOException {
+
+            response.setStatus(DefaultSpanManager.getInstance().current().getSpan() == null ? 500 : 204);
```

Comment:
Could you please add a tag to the current span and assert on that? (so all the tests are conceptually the same).